src/custom-element.js

```diff
@@ -40,6 +40,7 @@ import {
 import * as dom from './dom';
 import {setStyle, setStyles} from './style';
 import {LayoutDelayMeter} from './layout-delay-meter';
+import {ResourceState} from './service/resource';
```

Comment:
There's like 3-4 internal vars like this. Obviously having one state is probably easier here. So, this is ok, let's go with this. Alternatively, we can also set var in `unlayoutCallback` and reset it on the next layout - up to you.